ui/src/app/utils/index.tsx

```diff
@@ -438,8 +438,9 @@ export function formatWorkspaceResourceDisplayDate(time: string): string {
 }
 
 export function formatDomainString(domainString: string): string {
-  return fp.capitalize(domainString === Domain.PHYSICALMEASUREMENTCSS.toString()
-    ? Domain.PHYSICALMEASUREMENT.toString() : domainString);
+  return domainString === Domain.PHYSICALMEASUREMENTCSS.toString()
+    ? fp.capitalize(Domain.PHYSICALMEASUREMENT.toString())
+    : domainString === Domain.WHOLEGENOMEVARIANT.toString() ? 'VCF Files' : fp.capitalize(domainString);
```

Comment:
nit: if you move this ?: onto the next two lines, it'll be easier to tell what's going on / how many conditionals there are

Reply:
Done